src/Components/App/App.tsx

```diff
@@ -59,11 +86,11 @@ const App: React.FC = () => {
             </NavLink>
             </div>
           </div>}
-          />
-          <Route
+        />
+        {data.length && <Route
           path="/play"
-          element={<Continents countries={data}/>}
-        /> 
+          element={<Continents continents={data} assignSelections={assignSelections} />}
```

Comment:
The `assignSelections={assignSelections}` throws a TypeScript error for me. 

`Type '{ continents: CountriesData[]; assignSelections: (newSelection: string | object) => void; }' is not assignable to type 'IntrinsicAttributes & CountriesProps'.
  Property 'assignSelections' does not exist on type 'IntrinsicAttributes & CountriesProps'.ts(2322)`